challenge-OT4-B1/concordium-testnet-node-3d839245f8155da929b2029a1283d59f8910da51a31f7d453e592e46a2f98358.log

```diff
@@ -0,0 +1,3 @@
+{"PERSISTED_NODE_ID":"38dc0005419f494d","VERSION":"0.4.11"}
+runtime:0
```

Comment:
Unfortunatelly, I lost log file since Jan-21 till Feb-6 due to "out of space" issue. I guess I flushed it running the node again.